e2e/pnpm_workspace_deps/WORKSPACE

```diff
@@ -3,10 +3,6 @@ local_repository(
     path = "../..",
 )
 
-load("@aspect_bazel_lib//lib:repositories.bzl", "aspect_bazel_lib_dependencies")
-
-aspect_bazel_lib_dependencies()
```

Comment:
You missed a spot. I counted three when I greened up my other PR: https://github.com/aspect-build/rules_js/pull/1690/commits/28fdfa00b2df78bfcdd6a7f10df845efc030eed4

Reply:
This PR is green though. I don't think that 3rd one is required?

Reply:
Not required to green it up but it is a superfluous call to `aspect_bazel_lib_dependencies` in the 3rd case